src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -390,7 +391,9 @@ export async function install(
 
     const [releaseMeta, selectedChannel] = getReleaseMeta(gameInfo, channelName)
 
-    const releaseVersion: string | undefined = releaseMeta.name
+    
+    const releaseVersion: string | undefined = sanitizeVersion(releaseMeta.name)
```

Comment:
nit: we can remove the `string | undefined`

Reply:
:+1: fixed